test_upgrade_groups.yml

```diff
@@ -0,0 +1,19 @@
+- hosts: 127.0.0.1
+  connection: local
+  become: yes
+  vars_files:
+    - answerfile.yml
+  tasks:
+    - name: Update upgrade plan
```

Comment:
upgrade groups?, not sure about diff between upgrade plan and group, please correct.

Reply:
Done.